avocado/utils/genio.py

```diff
@@ -99,7 +99,7 @@ def ask(question, auto=False):
     if auto:
         log.info("%s (y/n) y" % question)
         return "y"
-    return raw_input("%s (y/n) " % question)
+    return input("%s (y/n) " % question)
```

Comment:
This will break Python 2.7. `input()` has different behavior in Python 3 when compared to Python 2.7.
In python 2.7, `input()` runs the input as a python expression, so user would have to make strings explicit:

```
>>> genio.ask('So?')
So? (y/n) y
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "avocado/utils/genio.py", line 102, in ask
    return input("%s (y/n) " % question)
  File "<string>", line 1, in <module>
NameError: name 'y' is not defined

>>> genio.ask('So?')
So? (y/n) 'y'
'y'
>>> 
``` 

Solution here is either use `six.moves.input` or
```
try:
   input = raw_input
except NameError:
   pass
```

Reply:
True! Thanks for catching this.  I'll be using `six.moves.input`.